build.gradle

```diff
@@ -49,6 +49,7 @@ ext {
 
 ext.libraries = [
         asyncHttpClient: 'org.asynchttpclient:async-http-client:2.12.3',
+        commonsLang3: 'org.apache.commons:commons-lang3:3.5',
```

Comment:
Do you think this one might need to be shaded?

Reply:
I don't think so. The breaking API changes in this library are fairly minimum.